Well-Coffee-UI/src/App.jsx

```diff
@@ -1,18 +1,111 @@
 import NavigationBar from "./components/navigation/NavigationBar";
-import NavigationRoutes from "./components/navigation/NavigationRoutes";
-import Footer from "./components/footer/Footer";
+import { useState } from "react";
+import { Route, 
+  Routes, 
+  Navigate } from "react-router-dom";
+import HomePage from "./components/pages/HomePage";
+import AccountPage from "./components/pages/AccountPage";
+import OrderPage from "./components/pages/OrderPage";
```

Comment:
Also need to reflect the right file: import OrderFormPage from "./components/orderForm/OrderFormPage";

Reply:
We could also just import the component into the OrderPage file.

Reply:
Resolved